string task_file_name
---
bool success
string serialized_task
